stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/service/sync/OfflineSyncFirebaseMessagingReceiver.kt

```diff
@@ -0,0 +1,14 @@
+package io.getstream.chat.android.offline.service.sync
+
+import android.content.Context
+import io.getstream.chat.android.client.logger.ChatLogger
+
+internal class OfflineSyncFirebaseMessagingHandler {
+    private val logger = ChatLogger.get("OfflineSyncFirebaseMessagingReceiver")
```

Comment:
File name and log tag ends with "Receiver". Was it done intentionally?

Reply:
Nice catch! Thanks